src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Figure implements FindArea {
+    private int biggerBaseSide;
+    private int smallerBaseSide;
+    private double side;
+    private int height;
+
+    public IsoscelesTrapezoid() {
+        super();
+        int firstRandomInt = new Random().nextInt(1, 10);
+        int secondRandomInt = new Random().nextInt(1, 10);
+        if (firstRandomInt > secondRandomInt) {
+            biggerBaseSide = firstRandomInt;
+            smallerBaseSide = secondRandomInt;
+        } else {
+            smallerBaseSide = firstRandomInt;
+            biggerBaseSide = secondRandomInt;
+        }
+        this.height = 10;
```

Comment:
Consider allowing the height to be a random value or a parameter to provide more flexibility in trapezoid dimensions.